### PR TITLE
Add translated NLU docs

### PR DESCRIPTION
**Pull request by jdicarl**
🚫: Waiting for the pt-br toc fix on staging and corresponding commit here before merging.
Issue open for BM doc team: https://github.ibm.com/Bluemix/Documentation-content/issues/998

Addresses issue #2776 from the Watson/developer-experience repo
https://github.ibm.com/Watson/developer-experience/issues/2776

- Add translated files for de, es, it, ja, and pt-BR
- Waiting for 5 more languages

### CODE REVIEW

**Comment by SirSpidey** [comment]
@jdicarl I don't see the TOC for pt-br in staging. Do you see it?

BTW, they have deployed a fix to the language switcher in dev-console. You can find it at the bottom of the page at https://dev-console.stage1.bluemix.net/docs/services/NaturalLanguageUnderstanding/getting-started.html

**Comment by jdicarl** [comment]
@SirSpidey Yes, I see the pt/BR toc file https://github.com/IBM-Bluemix-Docs/NaturalLanguageUnderstanding/pull/3/commits/d00c144d30766b8493c8586c4143fa67c770aae6#diff-10075411200696d6ad504ff2ab1bc4f4

**Comment by SirSpidey** [comment]
@jdicarl do you see the pt-br toc in the docs build? I don't.

**Comment by jdicarl** [comment]
@SirSpidey Thanks for the link to https://dev-console.stage1.bluemix.net/docs/services/NaturalLanguageUnderstanding/getting-started.html. I was trying to view it on staging yesterday and wasn't able to get any translations to show. I'll look into the problem and will make new commits when I figure it out.

File: nl/pt/BR/toc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        [Vídeos do Watson](https://www.youtube.com/user/IBMWatsonSolutions)
+    {: .navgroup-end}
+
+    {: .navgroup id="howto"}
+    [Docs expandidos](https://www.ibm.com/watson/developercloud/doc/natural-language-understanding/index.html)
+    {: .navgroup-end}
+
+    {: .navgroup id="reference"}
+    [Referência de API
+](https://www.ibm.com/watson/developercloud/natural-language-understanding/api/v1/)
```
Comment (jdicarl) [inline nl/pt/BR/toc]: Allen found this broken link. Remove the line ending after `API` on Line 23.

**Comment by SirSpidey** [comment]
@jdicarl I don't see it working on staging yet. Want to make sure it works before we push to prod.

**Comment by jdicarl** [comment]
I got the last language returned today, so I'm going to add a commit to this PR and do them all at once

**Comment by jdicarl** [comment]
Pushed to staging first so we could check the files. The PR: https://github.ibm.com/Bluemix-Docs/NaturalLanguageUnderstanding/pull/4

**Comment by SirSpidey** [comment]
@jdicarl just saw your comment. Not sure why they don't test languages in staging. I think we found a reason why that's not always a good idea ;-)

**Comment by jdicarl** [comment]
Closing because I did a PR to merge all the languages except pt/BR

**Pull request closed without merging**